networkx/algorithms/graph_hashing.py

```diff
@@ -172,25 +177,26 @@ def weisfeiler_lehman_subgraph_hashes(
     Lists of subgraph hashes are sorted in increasing order of depth from
     their root node, with the hash at index i corresponding to a subgraph
     of nodes at most i edges distance from u. Thus, each list will contain
-    ``iterations + 1`` elements - a hash for a subgraph at each depth, and
-    additionally a hash of the initial node label (or equivalently a
-    subgraph of depth 0)
+    `iterations` elements - a hash for a subgraph at each depth. If
+    `include_initial_labels` is set to `True`, each list will additionally
+    have contain a hash of the initial node label (or equivalently a
+    subgraph of depth 0) prepended, totalling `iterations + 1` elements.
```

Comment:
should I be using latex notation in this paragraph for i and u?

Reply:
The general guide I use is "test is better than latex except when it isn't". So symbols like i and u should be kept in text rather than LaTeX. Mathematical expressions can become unreadable in text format so that's when we should use LaTeX.